extensions/amp-analytics/0.1/cookie-writer.js

```diff
@@ -190,7 +190,9 @@ export class CookieWriter {
           // provide a way to overwrite or erase existing cookie
           if (value) {
             const expireDate = Date.now() + BASE_CID_MAX_AGE_MILLIS;
-            setCookie(this.win_, cookieName, value, expireDate);
+            setCookie(this.win_, cookieName, value, expireDate, {
```

Comment:
One question about the `highestAvailableDomain`. Would the old stored cookie be erased after the change?

Reply:
kind of. you will be able to read both values. It's up to the reader which value to pick.
```
> document.cookie = 'abc=1; domain=example.com'
"abc=1; domain=example.com"
> document.cookie = 'abc=2; domain=www.example.com'
"abc=2; domain=www.example.com"
> document.cookie
"abc=1; abc=2"
```

Reply:
I see, Thank you. Right now AMP doesn't provide a way to read cookies that are set in this way. But if we do in the future I believe we need to make sure to return the right cookie value. In that case what do you think we can do?

Reply:
a safe bet is to read the last value (I think it's ordered by freshness).

Reply:
Sounds good.